react/components/Variation/Items/MultipleChoice.js

```diff
@@ -51,7 +51,9 @@ class MultipleChoice extends Component {
     return (
       <div className="vtex-product-customizer__multiple-choice w-100 flex justify-between items-center">
         <div className="flex align-center">
-          <img src={item.image} width="48" className="br3 h-100" />
+          <div>
+            <img src={item.image} width="48" className="br3" />
```

Comment:
Hard coded size?

Reply:
It was already that way, I just wrapped it around a div